recipes/hub/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "hub" %}
+{% set version = "2.2.9" %}
+{% set sha256 = "b3f949c4500288a18ed68c38755962c9571c9e10063fb77583a19d0fcca5ecdf" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/github/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - go >=1.8
```

Comment:
Does this compile stuff? If so, you need `toolchain` here

Reply:
Sorry about that, I thought it was only needed for compiling with gcc. 

Either way this is no ready to merge.